src/main/webapp/gast/person.jsp

```diff
@@ -43,18 +43,25 @@
   <table id="personen-table" class="content-table">
 	<tbody>
 		<tr>
+                    <tr>
                     <th><% Language.printTextfield(out, session, "person", "Person"); %></th>
 
-			
+
 			<td><jsp:include page="../inc.erzeugeFormular.jsp">
 				<jsp:param name="ID" value="<%= id %>" />
 				<jsp:param name="Formular" value="person" />
 				<jsp:param name="Datenfeld" value="Standardname" />
 				<jsp:param name="size" value="50" />
 				<jsp:param name="Readonly" value="yes" />
-			</jsp:include>
+                            </jsp:include>
               <span style="float:right;display:block;font-weight:bold;">
-              </span></td>
+              </span>
```

Comment:
I think there could be a more efficient way of using empty space here:
- The span uses more space than it should
- The span is always generated, even if there is no GNDLink available
=> i guess this should be moved to the GNDLink specific logic in inc.erzeugeFormular.jsp

Reply:
ich würde es hier mal so lassen und im neuen Design dann verbessern, da dieser style sowiso verändert wird, können wir das hier lassen  bzw. übersehen

Reply:
so sieht das dann aus im neuen design und noch der code:
 <tr class="ut-table__row">
                    <th class="ut-table__item ut-table__header__item" scope="col">
                        <% Language.printTextfield(out, session, "person", "Person"); %>
                    </th>
                     <td class="ut-table__item ut-table__body__item">
                        <jsp:include page="../inc.erzeugeFormular.jsp">
				<jsp:param name="ID" value="<%= id %>" />
				<jsp:param name="Formular" value="person" />
				<jsp:param name="Datenfeld" value="Standardname" />
				<jsp:param name="size" value="50" />
				<jsp:param name="Readonly" value="yes" />
			</jsp:include>
                        <span style="margin-left: 5px;>
                            <jsp:include page="../inc.erzeugeFormular.jsp">
                                <jsp:param name="ID" value="<%=id%>" />
                                <jsp:param name="Formular" value="person" />
                                <jsp:param name="Datenfeld" value="GNDLink" />
                            </jsp:include>
                        </span>
                    </td>
                </tr>
![image](https://github.com/ubtue/neg/assets/105979344/3a2a103b-5efe-4cc7-a0b7-991ca01c365f)

Reply:
![image](https://github.com/ubtue/neg/assets/105979344/b292e149-4972-430e-abe6-b5aa83428de7)

Reply:
<table class="ut-table ut-table--striped ut-table--striped--color-primary-3">

            <tbody class="ut-table__body">

                <tr class="ut-table__row">
                    <th class="ut-table__item ut-table__header__item" scope="col">
                        <% Language.printTextfield(out, session, "person", "Person"); %>
                    </th>
                     <td class="ut-table__item ut-table__body__item">
                        <jsp:include page="../inc.erzeugeFormular.jsp">
				<jsp:param name="ID" value="<%= id %>" />
				<jsp:param name="Formular" value="person" />
				<jsp:param name="Datenfeld" value="Standardname" />
				<jsp:param name="size" value="50" />
				<jsp:param name="Readonly" value="yes" />
			</jsp:include>
                        <span style="margin-left: 5px;>
                            <jsp:include page="../inc.erzeugeFormular.jsp">
                                <jsp:param name="ID" value="<%=id%>" />
                                <jsp:param name="Formular" value="person" />
                                <jsp:param name="Datenfeld" value="GNDLink" />
                            </jsp:include>
                        </span>
                    </td>
                </tr>